testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/SinglenodeSparkIceberg.java

```diff
@@ -56,7 +57,12 @@ public SinglenodeSparkIceberg(Standard standard, Hadoop hadoop, DockerFiles dock
     @Override
     public void extendEnvironment(Environment.Builder builder)
     {
-        builder.configureContainer(HADOOP, container -> container.setDockerImageName("ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion));
+        builder.configureContainer(HADOOP, container -> {
+            container.setDockerImageName("ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion);
+            container.withCopyFileToContainer(
+                    forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/singlenode-spark-iceberg/apply-hive-config-for-iceberg.sh")),
+                    CONTAINER_HADOOP_INIT_D + "/apply-hive-config-for-iceberg.sh");
+        });
```

Comment:
This was removed in the first commit and then added back in the last one?

Reply:
a different sh file applying a different override. https://github.com/trinodb/trino/pull/9082/commits/68e442c1617600ea3ea0c40f4654613709691432#diff-b8c303745250261186fd02b08ceb61061de42d5736ee83a4ef4cb34d25846390L63